discord/abc.py

```diff
@@ -82,11 +82,11 @@ class User(metaclass=abc.ABCMeta):
 
     The following implement this ABC:
 
-    - :class:`User`
-    - :class:`ClientUser`
-    - :class:`Member`
+    - :class:`User <discord.User>`
+    - :class:`.ClientUser`
```

Comment:
In the entirety of this file you use the `.` which is a bandaid. A lot of these can be replaced with an explicit module:

`~discord.Class`.